content/release-notes/2023-05-16-storage-and-compute.md

```diff
@@ -0,0 +1,25 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Updated supported PostgreSQL versions to 14.8 and 15.3, respectively.
+- Compute: Neon uses compute endpoint domain names to route incoming client connections. For example, to connect to the compute endpoint `ep-mute-recipe-239816`, we ask that you connect to `ep-mute-recipe-239816.us-east-2.aws.neon.tech`. However, the PostgreSQL wire protocol does not transfer the server domain name, so Neon relies on the SNI (Server Name Indication) extension of the TLS protocol to do this. Unfortunately, not all PostgreSQL clients support SNI. When these clients attempt to connect, they receive an error indicating that the "endpoint ID is not specified".  
```

Comment:
it's `Proxy`, not `Compute` to be precise

Reply:
Both fixed. Thank you